clusterloader2/testing/density/config.yaml

```diff
@@ -54,6 +54,11 @@ steps:
       action: start
       nodeMode: {{$NODE_MODE}}
       resourceConstraints: {{$DENSITY_RESOURCE_CONSTRAINTS_FILE}}
+  - Identifier: Probes
```

Comment:
Can we enable in a separate PR? In case of problems it's smaller thing to revert then :-)

Reply:
Still - even if unused I would prefer to add it in a separate PR, you still register it, even if it doesn't perform any non-trivial code, it does something.

Reply:
Done